biteback/modules/temperature.py

```diff
@@ -16,9 +16,14 @@ class Temperature(module.BasicModule):
     final   = TempFinal()
 
     def run(self):
-        temp = float(shell("/etc/munin/plugins/temp").split(" ")[1])
-        if temp > 100.0:
-            return False
+        temp = shell("/etc/munin/plugins/temp").split(" ")[1]
+        if len(temp) > 0 and temp[0].isdigit():
+            # only convert if temp has at least a digit
+            temp = float(shell("/etc/munin/plugins/temp").split(" ")[1])
```

Comment:
I don't think you need to execute the shell again here.

Reply:
Nope, fixed.